components/desktop/libreoffice/Makefile

```diff
@@ -38,7 +38,7 @@ COMPONENT_ARCHIVE_HASH_1 = sha256:983941d31ee8d366085cadf28db75eb1f5cb03ba1e5853
 COMPONENT_ARCHIVE_URL_1 = https://dev-www.libreoffice.org/src/$(COMPONENT_ARCHIVE_1)
```

Comment:
Please add
COMPONENT_REVISION= 1
It is always needed to either change (increase) the COMPONENT_VERSION or set/increment the COMPONENT_REVISION.